frontend/app/controllers/Joiner.scala

```diff
@@ -170,11 +183,24 @@ trait Joiner extends Controller with ActivityTracking {
       Future.sequence(optFuture.toSeq).map(_.headOption)
     }
 
+    val futureContentOpt = request.cookies.get("GU_MEM_REFERER").map(_.value).map { referer =>
```

Comment:
Not sure if I should rename this…

Refe**rr**er is the correct spelling of the word but it's actually misspelled in the spec, so the HTTP header name is Refe**r**er. Guess it makes sense to keep it consistent with the HTTP header we are checking.